browser/js/adminDash/song/adminDash.songs.html

```diff
@@ -1,7 +1,7 @@
 <ul>
   <span ng-repeat="song in songs">
-    <h3>{{song.title}}</h3>
-    <button ui-sref="songForm">Edit</button>
+    <h3><a ng-href="/songs/{{song.id}}">{{song.title}}</a></h3>
```

Comment:
Not sure exactly what the difference is but ng-href works for linking to the single-product view here and ui-sref doesn't seem to be working.